### PR TITLE
Installation changed

### PR DESCRIPTION
**Pull request by pconesa**
Test added: scipion3 tests emready.tests.test_protocol_sharpening.TestEMReadySharpening Call done using conda activation env variable

### CODE REVIEW

**Comment by pconesa** [comment]
Installation is working for me fine now after some tweaks.

But the test fails in my case:

numpy.core._exceptions.MemoryError: Unable to allocate 57.9 GiB for an array with shape (140608, 48, 48, 48) and data type float32

I do not have 58GB in my laptop.

File: emready/tests/test_protocol_sharpening.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            emReadyProtocol = self.proj.newProtocol(ProtEMReadySharppening,
+                                              objLabel='EMReady %s' % label,
+                                              in_vol=vol)
+            self.proj.launchProtocol(emReadyProtocol)
+
+            outputVol = getattr(emReadyProtocol, EMReadyOutputs.sharpenedVolume.name)
+            self.assertIsNotNone(outputVol,
+                                 "Sharpened volume is None for %s test." % label)
+
+            self.assertEqual(self.inputVol.getDim(),
```
Comment (JiahuaHe) [inline emready/tests/test_protocol_sharpening.py:100]: The size of volume might be changed after applying EMReady

File: emready/tests/test_protocol_sharpening.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                             outputVol.getDim(),
+                             "sharpened Volume has different size than inputVol "
+                             "for %s test" % label)
+
+            self.assertEqual(self.inputVol.getSamplingRate(),
```
Comment (JiahuaHe) [inline emready/tests/test_protocol_sharpening.py:105]: The sampling rate of volume is unified to 1.0 angstrom after applying EMReady

File: emready/tests/test_protocol_sharpening.py (hunk 1 above)
Comment (JiahuaHe) [inline emready/tests/test_protocol_sharpening.py]: The sampling rate of volume will be interpolated to 1.0 angstrom and therefore the size of volume might be changed after applying EMReady. These two assertions should be removed.

File: emready/tests/test_protocol_sharpening.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
+# ***************************************************************************
+# *
+# * Authors:     David Maluenda (dmaluenda@cnb.csic.es) (2018)
```
Comment (JiahuaHe) [inline emready/tests/test_protocol_sharpening.py]: Is author name correct?

**Review by JiahuaHe — CHANGES_REQUESTED** [review]

**Review by JiahuaHe — APPROVED** [review]

File: emready/protocols/protocol_emready_sharpening.py [hunk 4]
```diff
@@ -120,8 +120,8 @@ def createOutputStep(self):
 
     def _validate(self):
         errors = []
-        if (40 < self.stride or self.stride < 10):
-            errors.append('`stride` should be within [10, 40].')
+        if (48 < self.stride or self.stride < 10):
```
Comment (JiahuaHe) [inline emready/protocols/protocol_emready_sharpening.py]: The lower limit of `stride` is 12

**Review by JiahuaHe — CHANGES_REQUESTED** [review]

**Review by JiahuaHe — APPROVED** [review]

File: emready/protocols/protocol_emready_sharpening.py [hunk 5]
```diff
@@ -120,10 +120,10 @@ def createOutputStep(self):
 
     def _validate(self):
         errors = []
-        if (48 < self.stride or self.stride < 10):
-            errors.append('`stride` should be within [10, 48].')
+        if (12 < self.stride < 48):
```
Comment (JiahuaHe) [inline emready/protocols/protocol_emready_sharpening.py]: It seems that errors will be reported for stride between 12 and 48, which is just in contrast to its original purpose (to make sure 12 <= stride <= 48).

**Review by JiahuaHe — CHANGES_REQUESTED** [review]

**Review by JiahuaHe — APPROVED** [review]

**Pull request merged**